source/Personality.cpp

```diff
@@ -49,6 +49,7 @@ namespace {
 	const int TARGET = (1 << 25);
 	const int MARKED = (1 << 26);
 	const int LAUNCHING = (1 << 27);
+	const int MERCIFUL = (1 << 28);
```

Comment:
This list has the same order as the other lists in the file, so move this after OPPORTUNISTIC.